library/src/hooks/autocomplete.ts

```diff
@@ -17,21 +17,21 @@ export const useAutocomplete = (
 ): google.maps.places.Autocomplete | null => {
   const {inputField, options, onPlaceChanged} = props;
   const placeChangedHandler = useRef(onPlaceChanged);
-  const {map} = useGoogleMap();
+  const {googleMapsAPIIsLoaded} = useContext(GoogleMapsContext);
```

Comment:
III :D

Do you mean:
```suggestion
  const {googleMapsAPIsLoaded} = useContext(GoogleMapsContext);
```?

Same for the ones below.

Reply:
Got to say, it is okay for now, but it does not look very beautiful with the `useContext(GoogleMapsContext)`. So, let's see if we can adjust it one day :D

Reply:
No `googleMapsAPIIsLoaded` was intended. 🤔 "google Maps API Is Loaded"

Reply:
Oh wow... that's hard to read. What about just `googleMapsAPILoaded`

However, not wrong like this. So fine with it :)